### DISCUSSION TITLE
TypedSql - how does the /sql folder work with multiple connections?

### DISCUSSION THREAD

**Discussion by PatricioNG** [prisma/prisma#25105] [Q&A]
### Question

I've got multiple connections set up to different databases, in all my scripts I'm passing `--schema xxx` for segmenting out the proper connection and prisma client, how does this work with the `/sql` folder? 

For example:
![image](https://github.com/user-attachments/assets/bc622b18-f361-40a8-a9be-7e09587c026a)

I only want a `/sql` folder that would be for raw queries in my `OMS` connection, is that possible at this stage?

**Comment by ludralph** [comment]
Hi @PatricioNG 

Can you please share this question in the [feedback](https://github.com/prisma/prisma/discussions/25106) for typedSQL?
